Makefile

```diff
@@ -21,36 +21,44 @@
    SMP_FLAGS = -j$(shell nproc)
  endif
 
- ISYS_ROOT = $(shell ./bootstrap/coriolisEnv.py --query-isys-root)
- INST_ROOT = $(shell ./bootstrap/coriolisEnv.py --query-inst-root)
- WORK_ROOT = ${HOME}/coriolis-2.x/Linux.el9/Release.Shared
+ ISYS_ROOT    = $(shell ./bootstrap/coriolisEnv.py --query-inst-root)
+ SRC_ROOT     = $(shell ./bootstrap/coriolisEnv.py --query-src-root)
+ BUILD_ROOT   = $(ISYS_ROOT)/build
+ INST_ROOT    = $(ISYS_ROOT)/install
```

Comment:
i'd set BUILD_ROOT to be $CWD/builddir
INST_ROOT should probably default to /usr/local

Reply:
I don't like to put builded files inside the source tree. For me ``$CWD/builddir`` is not *out of source* build. Even if it seems many people do it.

Reply:
So for standard makefile behavour, you default to /usr/local for install, and override that with the PREFIX variable.

I understand that keeping this Makefile around is for exiting users inside LIP6, but at the moment i'm conerned its just going to be a confusing artefact for others. Not sure how to best resolve this

Reply:
Not only. It is for the development process. I see no problem in changing that if it is more standard and fit my specifications:

1. The build/install process must be fast and easy, ideally done with one command.
2. Two installations must cohabit : release and debug, and it should be easy to switch between them (this one of the task of ``coriolisEnv.py``).
3. The installed versions must be easily usable (PATH, LD_LIBRARY_PATH, PYTHONPATH). This is the task of ``coriolisEnv.py``